index.rst

```diff
@@ -42,17 +42,23 @@ Avro schemas can be composed of nested sub-schemas under a top level namespace.
 Nesting simplifies what would otherwise be monolithic schemas as new fields are added.
 For example, the base alert schema (``lsst.alert``) is of type "record" and includes previous detections of DIA sources as an array of type ``lsst.alert.diaSource``.
 
-The current schema proposed for use with LSST alerts is stored in the `lsst/alert_packet`_ repository.
-This contains all fields specified by the LSST Data Products Definition Document (LSE-163; :cite:`LSE-163`).
+The current schemas contain all fields specified by the LSST Data Products Definition Document (:cite:`LSE-163`).
 At this stage in construction, this schema should be regarded as exploratory and subject to rapid change; as we move closer to the operational era, a change control process will be implemented.
 
+The current schema proposed for use with LSST alerts is stored in the `lsst/alert_packet`_ repository.
+This primary purpose of this package is to enable generation of alerts during Prompt Processing.
+Versioning of this package with the rest of the Science Pipelines software ensures reproducability.
+During operations, users will access the production schemas using other tools (see :ref:`alertManagement`).
+
 .. _lsst/alert_packet: https://github.com/lsst/alert_packet
 
+.. _alertManagement:
+
 Management and Evolution
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
 Avro packets may be shipped with or without an embedded schema.
```

Comment:
This line is a little confusing. It sounds like a normative statement about what the project will do. I think it's intended to actually say "embedded schemas are technically possible in Avro but not required", or something?

Anyway, it's confusing and could use an edit.

Reply:
Understood--I took another crack at it.